internal/proto/controller/api/resources/aliases/v1/alias.proto

```diff
@@ -60,7 +72,8 @@ message Alias {
     (custom_options.v1.mask_mapping) = {
       this: "value"
       that: "Value"
-    }
+    },
+    (google.api.field_behavior) = REQUIRED
```

Comment:
Yeah this is unfortunate. I don't think there's any way to have this only apply to create, but in our initial feedback session not knowing what is and isn't required was a big problem. Adding this annotation adds a nice `required` icon to the field, but as you say it will also show on the UpdateAlias request.
![image](https://github.com/hashicorp/boundary/assets/6604151/7a4c92be-13b9-493e-9599-8830b6eefee3)
Do you think it's better to not have the `required` annotation at all to avoid confusion in the UpdateAlias request?

Reply:
I've removed them